### DISCUSSION TITLE
While playing, a glitch where the shortcut is used by going back to the desktop screen and pressing a key of the alphabet (probably the windows key is always pressed by a bug).

### DISCUSSION THREAD

**Discussion by mushiseiya** [ppy/osu#27220] [Q&A]
### Type

Other

### Bug description

While playing, a glitch where the shortcut is used by going back to the desktop screen and pressing a key of the alphabet (probably the windows key is always pressed by a bug).
When I play mania, it takes me back to the desktop screen.
The keys used in 4K are s,d,k,l, so Win+s brings up the start menu and Win+k brings up the cast function.
Pressing the windows key will release the hold down.
ver.Lazer 131,0 
Windows 10 Home
22H2
‎2020/‎12/‎02
19045.3930
Windows Feature Experience Pack 1000.19053.1000.0
IME:Japanese 

### Screenshots or videos

https://github.com/ppy/osu/assets/86839205/2e5f89b8-d24f-44c1-9d8d-2e2a282a2fc4

### Version

Lazer 131,0 

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14266829/compressed-logs.zip)

```[tasklist]
### Tasks
```

**Comment by frenzibyte** [comment]
I would recommend you check that this is not an issue with your keyboard. In addition, check whether you have `Disable Windows key during gameplay` enabled in settings.

**Reply by mushiseiya** [reply]
![image](https://github.com/ppy/osu/assets/86839205/741976a3-8945-416d-a94e-2dbcc021c1ce)

Already checked off the list. Thanks for the reply.
